packages/blockchain-extension/extension/commands/exportSmartContractPackageCommand.ts

```diff
@@ -39,7 +39,14 @@ export async function exportSmartContractPackage(packageTreeItem?: PackageTreeIt
             }
             packageToExport = chosenPackage.data;
         }
-        const defaultPath: string = path.join(os.homedir(), `${packageToExport.name}@${packageToExport.version}.cds`);
+
+        let packageName: string = packageToExport.name;
+
+        if (packageToExport.version) {
```

Comment:
In v2 it's possible to create a package without a version?

Reply:
yes in v2 packages don't have versions. We are using the version property to name the file but if the package was created outside of vscode then it might not have the same format filename